service/src/main/java/bio/terra/cbas/controllers/MethodsApiController.java

```diff
@@ -293,6 +328,21 @@ private void createNewMethod(
             cbasContextConfig.getWorkspaceId());
 
     methodDao.createMethod(method);
+
+    GithubMethodDetails sourceDetails;
+
+    if (githubMethodSourceDetails != null) {
+      sourceDetails =
+          new GithubMethodDetails(
+              githubMethodSourceDetails.repository(),
+              githubMethodSourceDetails.organization(),
+              githubMethodSourceDetails.path(),
+              githubMethodSourceDetails.isPrivate(),
+              githubMethodSourceDetails.methodId());
```

Comment:
Maybe I am missing something, but isn't `sourceDetails` same as the parameter `githubMethodSourceDetails` passed into the function? Can `githubMethodSourceDetails` directly be used in L343?